ngrinder-controller/src/main/java/org/ngrinder/infra/config/Config.java

```diff
@@ -262,6 +265,26 @@ protected void copyDefaultConfigurationFiles() throws IOException {
 		home.copyFrom(resources);
 	}
 
+	private void copyScriptTemplates() {
+		File templateResourceDirectory = new File(getClass().getResource("/script_template").getFile());

```

Comment:
It is not working when running on executable jar
https://stackoverflow.com/a/25873705/8074178

I should use Inputstream to copy the folder or consider other ways.